vs-code/vscode.md

```diff
@@ -48,6 +49,28 @@ Most software is written using text editors. Visual Studio Code (VS Code) is a v
 
 [Slides: Local Development with VS Code](https://docs.google.com/presentation/d/1QAMxrS1ZOvtSn7MuYxTmeanrMQi_O8Z5UVq-4LzLqso/edit?usp=sharing)
 
+### VSCode Tips
+
+- **Auto Indent** - VS Code package to format your JavaScript / TypeScript / CSS using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
```

Comment:
I think there is an auto indent option on the menu and it is handy as well. Might be good if you add those instructions as well

Reply:
Thank you for pointing out. Added.